applications/nrf_desktop/doc/power_manager.rst

```diff
@@ -1,7 +1,132 @@
-.. _power_manager:
+.. _nrf_desktop_power_manager:
 
 Power manager module
 ####################
 
 Use the ``power_manager`` module to reduce the energy consumption of battery-powered devices.
 This is achieved by switching to low power modes when the device is not used for a longer period of time.
+
+
+Module Events
+*************
+
+
++----------------------------------+------------------------+----------------------------------+----------------------+------------------------------------+
+| Source Module                    | Input Event            | This Module                      | Output Event         | Sink Module                        |
++==================================+========================+==================================+======================+====================================+
+|                                  |                        | :ref:`nrf_desktop_power_manager` | ``power_down_event`` | :ref:`nrf_desktop_power_manager`   |
```

Comment:
Some interactions are missing here.

Reply:
yes - you are generating the table so its pointless to fill everything, right ;)